adapters/Unity/UnityAdapter/GADMAdapterUnityRewardedAd.m

```diff
@@ -37,6 +34,8 @@ @interface GADMAdapterUnityRewardedAd () <GADMAdapterUnityDataProvider, UnityAds
   BOOL _isLoading;
 }
 
+// The completion handler to call when the ad loading succeeds or fails.
+@property (nonatomic, copy)GADMediationRewardedLoadCompletionHandler adLoadCompletionHandler;
```

Comment:
Is there a reason why this was changed to a property?

Reply:
the intent is to access within the complete block.

Reply:
I believe we can create a strong reference to the adapter within the block in order to access `adLoadCompletionHandler` as an instance variable.

```
GADMAdapterUnityRewardedAd *__weak weakSelf = self;
UnitySingletonCompletion completeBlock = ^(UnityAdsError *error, NSString *message) {
  GADMAdapterUnityRewardedAd *strongSelf = weakSelf;
  if (!strongSelf) {
    return;
  }

  if (error) {
    NSError *errorWithDescription = GADUnityErrorWithDescription(message);
    strongSelf->_adLoadCompletionHandler(nil, errorWithDescription);
    return;
  }
  [UnityAds addDelegate:strongSelf];
  [UnityAds load:[strongSelf getPlacementID]];
};
```